apps/shopify/shopify.star

```diff
@@ -0,0 +1,554 @@
CEqkWQ3FQAAEQOA9AiTiQtCGSaLnk+j838PipiOMALjJmyls2dhS9SmlFKcSppkpLuMUCIAACFxAgOcTvcqbka0IlTcdvOAiTriGAEYAXOPK1IbUVkR/YxS1l3L2wAd5c6GzqUvhLAiAAAgMEOji34q1Je+8bxY6f/ffERgBcL+PBy1saFn8QZLxb3G6zjsHT+IABEDA8wTMSX686dgzWqsHeLj/qOeBeAQAAgCPODrZzLrWqoDQ6jEe4rsu+TyOQQAEvEeAO/9XlJD3B8sbf+09671tMV4BeND/YX801iG7b+Cg/36e4fuuBxHAZBAAASn+zM/9Kzv2zZmNzt+btwNGALzp90Grf9RcMaPfV7CeiL4khTQGL+AABEDAlQT4ib9HSfr+mT761gNVTfzOHx+vEkAA4FXPD7F7U6zy6oRQ6/n07UMu4SsIgIBbCGi5M+FLrFld1nTELSbBjrETQAAwdnaurFkfq5rPs4C/x8Zd70oDYRQIeJCAuayPhHE/1vN70PkjmIw5ACPA8eKlUCDaXNxRNFNIupdzBxz1IgPYDAKuIUDiCJG8u+OF0hvR+bvGq1kzBCMAWUPpvoaefvr2whPFZ74ohH4IaYXd519Y5F4CPKfnhJTqexNF96P3BFp73GspLBsPAQQA46HnkbrfjVZefHGhUStIP8iBwFSPmA0zQcBxBLjjP83r+evj3erhNQsbTznOAChsKQEEAJbidrawf31xSVFvf38IgYCz/Qjt3UfgfMcvfL4NoZt2nXSfhbAoFwQQAOSCqsvbTAoEvsYjAu93ubkwDwRsSwAdv21d4wjFEAA4wk32VPKR2NKpE0TPfQkp1igS0+ypJbQCAVcSeIczeT6q+hOPrVzQjGRernRx7o1CAJB7xq6X8GTMP+EsTVzGKwe+zjfUx1xvMAwEgXwRIDrOSfs3+eLxR9Dx58sJ7pGLAMA9vsy7JevWrVMl/gOLpKAHBYm5eVcICoCAWwiQOMy7eW5MdKvHeXJfr1vMgh35JYAAIL/8XSu9LlY9T2r6Ku88uITnCSDfhGs9DcNyTCDGT/yPhMoad/Lsft66Ax8QyB4BBADZY4mWUhCob170EfIl7uOkQsslyUkpiuAUCIBAEgEtqU+R/JUm9f3a8t0Hki7hEASySgABQFZxorHhCDy2r6bEFxdBUhSWJC4ZrhzOg4BXCZCQb3Og/Hhhor9hxfwWPsYHBHJLAAFAbvmi9SEEBrILXnL6FqHFl3k8s4KHNXEPDmGEr14jIA/y4P7mngmTt35l7rZur1kPe/NHAD+++WPvecn1e6s/xisH7uHNh5YzjOmeBwIAniHAwe+7HPz+W0KqjavLdv/OM4bDUFsRQABgK3d4Uxkz1fCkQmVuQ3wvPwnNw6iAN+8Dt1tNJHg+n95HZPzY1+V7ZuVndpx1u82wz94EEADY2z+e066urfJyqY3PC6KV/HLgCs8BgMGuI0BCH+M3XT/jp/4nwoHoYdcZCIMcSwABgGNd527FzZwCM27ev0BLcSffpEvZ2ovdbTGscxOBgRS9Qj6nlN56vHVuC9/P2k32wRZ3EEAA4A4/utqKgUyDcuICSXQnL5G6hZdIFbraYBjnSAI8wp/gpa4xns2/Vcju7eFA6xlHGgKlPUMAAYBnXO0OQzc2V0w3jILbeP+BZRwQlPHQquEOy2CFIwmQiLPeraTk0xf16GeXV0f/4kg7oLQnCSAA8KTb3WH0E5Gqab2FajHnFbhdG7oSIwPu8KvdrRh40hdyP//dVpiIP4U1+3b3GPQbjgACgOHI4LyjCAzsTCi7P0OkbiHSlVLKIkcZAGXtTUDSKZ7DH2Ulnzfi8R3YiMfe7oJ26RFAAJAeJ5RyEAFONmR0lJwu5dTpiznh0BKh5McdpD5UtQsBKf7EqjTzO/2dJe2To8uWbeuzi2rQAwSyQQABQDYoog1bE9gUq7w6IYxqkgmeSKjKWFmsKLC1x/KkHIkzvPS0jYf291DCaKydv/sPedIEYkHAEgIIACzBDCF2IWCmIn6n5PRcTspSyeuyF3AmwhswkdAu3rFWj3Pv8sVv+EewiV8d7Sk+MfllPOVb6wNIyy8BBAD55Q/peSZQF/NPluKiOTzMO48nE35aK7oZkwnz7JQciT/X4avfkqRf8w/fi4U91IJZ+zmCjWYdQQABgCPcBCWtImAGBEJPnMtDwfOkotk8UjCb/5NMsUo+5GSRAFGnkPIAj/Ts5/kgL3b10csPVDV1ZVECmgIBRxNAAOBo90H5XBMwMxJeOu/A1dqnZ/OmRXN4rwIzKLiG9ysoyLVstJ8+AfZJP3fyr3Gy/QNca79QxoHQzbveYD9x/48PCIBAKgIIAFJRwTkQGIHApldmFvSfKv6oFGqmUDSTXx3M5G7meh41mDxCNVzKFoGByXryTd4v4hAP5x/kwOxg70VFB7GVbrYAox2vEEAA4BVPw86cE/hh04IPGAXqGn7kvJbzEFzDQcG1nBP+b5GTYKzoqZf3gvgjz8l4jZke4qf517SmQyf2lb6O3PpjZYp6IPAeAQQA77HAEQhknQAPTcv6fZWXcRKZq3ik4EoWcJUQ6ipB+kru1K70fHBgJtjR4k+cQ/+wVuIwj9j/kVdlHO7nf6v9jccwhJ/1WxINgsAgAQQAgyhwAALWE6h/YdH7E5ou8+n4h3hbg8s5KLhMaLpcSvqAJjWDXyuUSNIlPJlNWa/dOCRyOkaSqoNHQTqU1G8TyT9zQqa3OPB5izv5/40r39G4LnxrbeC5znFIQVUQAIFxEEAAMA54qAoCVhA4l9mwq4SfkEt8cTL/TpNCTyWhpir+mxByKv9HniqEnsSJjjgFMk0QUk3k1w8Xc9hQyJ3wFH5fnhRAnLv+/3Qn3c3v0nsGz0mpOfh4l7To41GKLl5Cd5ZHMHpJ6tM8gnGWA5VOQ1CnFoo7cDqpzBn3pE7GfbJDadHR0fZ37RimH6SJAxCwJYH/Az7x75xxnfKnAAAAAElFTkSuQmCC
```

Comment:
To save some space, here's the same graphic pre-scaled to 24x24 with exif data removed:
```
iVBORw0KGgoAAAANSUhEUgAAABgAAAAYCAMAAADXqc3KAAAABGdBTUEAALGPC/xhBQAAACBjSFJNAAB6JgAAgIQAAPoAAACA6AAAdTAAAOpgAAA6mAAAF3CculE8AAABI1BMVEUAAACWv0iWv0iWv0iWv0iWv0iWv0iWv0iWv0iWv0iWv0iWv0iWv0iWv0iWv0iWv0iWv0iWv0iWv0idw1TP4qzi7s3O4aqfxVjS5LGz0XvN4ajS5LDP4quix16tzW+81ovT5LK10n/q8tr2+vD+/v3D25epymmcw1PZ6L3f7Mj8/fr////9/vyZwU251YanyWXX57mvznSqy2rp8tm204Dd6sPX5rmkyGGgxVn6/PbG3Jvq8tuxz3emyWTV5bb4+/K304K+147l79Hy9+imyWPG3Jz0+Oyqy2v2+e/3+vHo8divznP///6bwlCcwlKkx2DF3Jqvz3TC2pWy0HjY57zs8979/vvJ3qGqy2zS47Dm79P5+/Tf7MfI3qCx0HibwlGXv0nUrRiRAAAAEnRSTlMAInq75vu8IKn9IVzzdv51/HlGNAFfAAAAAWJLR0QrJLnkCAAAAAFvck5UAc+id5oAAAEiSURBVCjPdVLnWgJBDMw17jiQsjZU7ChqBHsBC6BS7Nh7ef+n8JKscvh9zJ/cztwm2UkABIZp2Y5jR0wDwnC9qNKIen6Hj8VVCPHYL9+XUF1IJPX/wvcPDA4Na4Xv+DpPZmR0LDuus1EdT74nJqemZ2Zzc/N88oI+pZ/8wuISEpYLdEwZYDJfFJawwoQJEY5ZxNW19Q0SNpmwwOa4hdtqZ7dEQpkJGxyOe7h/gDlOdciEo4UKVlVN6hxpgVPlEY9PVJ2Fhk5lUWhi6/RMnbNwcSnFud0qZq6uVVv6bRKTlge2sVK+uS2JUCTzDbHk7v7h8e+FT2IJuGJi4bn28vrWev/4/ApMdEO2E77DtgMkewyq92iDYXmpzjK4XXvip/+tzw/wsDO/5t4LZQAAAABJRU5ErkJggg==
```